specification/domainservices/resource-manager/Microsoft.AAD/2017-01-01/domainservices.json

```diff
@@ -26,24 +26,30 @@
       }
     }
   },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
   "paths": {
     "/providers/Microsoft.AAD/operations": {
       "get": {
         "tags": [
-          "DomainServices"
+          "Operations"
         ],
-        "description": "Lists all the available Domain Services operations.",
-        "operationId": "DomainServiceOperations_List",
-        "parameters": [
-          {
-            "$ref": "#/parameters/ApiVersionParameter"
-          }
+        "summary": "Operations_List",
+        "description": "Gets the details of all operations possible on the Microsoft.AAD resource provider.",
+        "operationId": "Operations_List",
+		"produces": [
+          "application/json"
         ],
         "responses": {
           "200": {
-            "description": "HTTP 200 (OK) if the operation was successful.",
+            "description": "The operation succeeded. The response contains the possible operations on the Microsoft.AAD resource provider.",
             "schema": {
-              "$ref": "#/definitions/OperationEntityListResult"
+              "$ref": "#/definitions/OperationListResult"
```

Comment:
This is definitely an SDK breaking changes. Is there a significant reason to do these breaking changes in the old version of API?